packages/client-redux/test/getActionType.test.js

```diff
@@ -0,0 +1,75 @@
+import { getActionType } from '../src/getActionType';
+
+describe('getActionType', () => {
+  it('should return action type - 3 parameters', () => {
+    function testFn(payload, options, dispatch = () => null) {
+        return dispatch({
+            resource: 'resource',
+            method: 'get',
+            path: 'path',
+            payload,
+            options,
+          });
+      }
+    const x = getActionType(testFn);
```

Comment:
get rid of the x

Reply:
sure
changed to a name which makes sense